metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

```diff
@@ -136,10 +138,22 @@ public Builder includeMeterFields(Set<String> fields) {
             return this;
         }
 
+        /**
+         * Map measurement to a defined measurement name, where the key is the measurement name
+         * and the value is the reqex the measurement should be mapped by.
+         *
+         * @param measurementMappings
+         * @return {@code this}
+         */
+        public Builder measurementMappings(Map<String, String> measurementMappings) {
```

Comment:
I like the idea of sending in the map with pattern values. Then we don't need to construct another map here and the caller should be sure, that they are sending in patterns that compile too.